tools/fetch-ieee.sh

```diff
@@ -23,30 +23,43 @@ while true; do
     esac
 done
```

Comment:
Not, it's not necessary. Rest of the changes look fine: I'll merge them after you remove the prompt.